packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-service.ts

```diff
@@ -41,6 +41,8 @@ export = {
         DesiredCount: 1,
         LaunchType: LaunchType.FARGATE,
         LoadBalancers: [],
+        EnableECSManagedTags: true,
+        PropagateTags: PropagateTagsFromType.SERVICE,
```

Comment:
The fact you had to introduce these means you're making a breaking change. We don't want that.